Header header
float32 vertical_speed
float32 roll
float32 pitch
float32 yaw
uint8 mode
uint8 engine_mode
